src/open_inwoner/cms/cases/views/services.py

```diff
@@ -112,26 +115,36 @@ def get_submissions_for_api_group(
 
     def get_submissions(self) -> list[SubmissionWithApiGroup]:
         all_api_groups = list(
-            ZGWApiGroupConfig.objects.exclude(form_service__isnull=True)
+            ZGWApiGroupConfig.objects.select_related(
+                "form_service",
+            ).exclude(form_service__isnull=True)
```

Comment:
I would expect:

```
all_api_groups = list(
            ZGWApiGroupConfig.objects.filter(form_service__isnull=False).select_related(
                "form_service",
            )
        )
  ```
  although Django might create the same SQL for both so it's just a cosmetic issue (?)